src/main/resources/org/broadinstitute/dsde/sam/liquibase/changesets/20221115_azure_managed_resource_group.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="dvoet" id="azure_managed_resource_group">
+        <createTable tableName="SAM_AZURE_MANAGED_RESOURCE_GROUP">
+            <column autoIncrement="true" name="id" type="BIGINT">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="tenant_id" type="VARCHAR">
+                <constraints nullable="false"/>
+            </column>
+            <column name="subscription_id" type="VARCHAR">
+                <constraints nullable="false"/>
+            </column>
+            <column name="managed_resource_group_name" type="VARCHAR">
+                <constraints nullable="false"/>
+            </column>
+            <column name="billing_profile_id" type="VARCHAR">
+                <constraints nullable="false" unique="true"/>
+            </column>
+        </createTable>
+        <addUniqueConstraint tableName="SAM_AZURE_MANAGED_RESOURCE_GROUP"
+                             columnNames="tenant_id,subscription_id,managed_resource_group_name"/>
+    </changeSet>
+</databaseChangeLog>
```

Comment:
Not a fan of xml migrations is there any way we can make them yaml or something more readable just a thought

Reply:
I don't really care but I try to stay consistent unless you want to make a decree that from this point forth, yada yada